src/Common/src/Interop/Unix/libssl/Interop.OpenSsl.cs

```diff
@@ -74,20 +74,20 @@ internal static bool DoSslHandshake(SafeSslHandle context, byte[] recvBuf, int r
                 BioWrite(context.InputBio, recvBuf, recvOffset, recvCount);
             }
 
-            libssl.SslErrorCode error;
-            int retVal = libssl.SSL_do_handshake(context);
+            Ssl.SslErrorCode error;
+            int retVal = Ssl.SslDoHandshake(context);
```

Comment:
I see a couple more redundant `Ssl.` qualifiers, I'm not going to point them all out (particularly because it would be annoying to do so if they were needed :smile:)